### DISCUSSION TITLE
Using Bubble Card as Nav causes a Double Click when New Page opened

### DISCUSSION THREAD

**Discussion by templeowls21** [Clooos/Bubble-Card#832] [General]
Kinda weird....I have a bubble card that navigates to a different page when tapped. 

On mobile, when I tap it, it will do another tap on the new page in the same location, which isn't good.

I've replaced the bubble with a both mushroom and button card, and the issue does not happen. Any suggestions? I'm trying to keep the same aesthetic throughout so I'd prefer to use a bubble

**Comment by Clooos** [comment]
Hi, I've worked on the tap actions today to make a better and lighter system that handle all the actions. With some luck this will fix your issue 🤞

**Reply by templeowls21** [reply]
Okay, thanks! Do I need to update anything in HA for your changes to take effect?

**Reply by Clooos** [reply]
Just wait for the next Bubble card beta 👌

**Reply by templeowls21** [reply]
Oh gotcha. Thanks!!
